internal/server/v1/propertyvalues/property_values.go

```diff
@@ -0,0 +1,286 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package propertyvalues
+
+import (
+	"container/heap"
+	"context"
+	"log"
+
+	cbt "cloud.google.com/go/bigtable"
+	pb "github.com/datacommonsorg/mixer/internal/proto"
+	"github.com/datacommonsorg/mixer/internal/server/pagination"
+	"github.com/datacommonsorg/mixer/internal/store"
+	"github.com/datacommonsorg/mixer/internal/store/bigtable"
+	"github.com/datacommonsorg/mixer/internal/util"
+	"google.golang.org/protobuf/proto"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// Prophet generates cache size at 500.
+// This (approximately) allows one request to be fulfilled by reading one page
+// from all import groups after merging.
+var defaultLimit = 1000
+
+type processor struct {
+	property string
+	entity   string
+	limit    int
+	// CursorGroup that tracks the state of current data.
+	cg *pb.CursorGroup
```

Comment:
nit: Since this is not local variable, I feel we could make the variables a bit longer / more representative. Same for "h" and "raw" below, etc.